src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import Logo from '../../Images/planet.png';
+
+function NavBar() {
+  return (
+    <div className="NavBar">
+      <div className="LogoContainer">
+        <img className="planet" src={Logo} alt="" />
+        <h1>Space Traverlers Hub</h1>
+      </div>
+      <div className="NavLinks">
+        <NavLink to="./Rockets.js" end className={({ isActive }) => (isActive ? 'currentPage' : undefined)}>Rockets</NavLink>
+        <NavLink to="./Missions.js" end className={({ isActive }) => (isActive ? 'currentPage' : undefined)}>Missions</NavLink>
+        <NavLink to="/" end className={({ isActive }) => (isActive ? 'currentPage' : undefined)}>My Profile</NavLink>
+      </div>
+    </div>
```

Comment:
- [ ] I know you are not using pure HTML now, but the best practices such as using accessible tags still apply! Please consider using `<header>` and `<nav>` tags for the navigation toolbar! 🤓 🔧 

- [ ] Also, it would be great if you add an alternative text `alt` to your logo! Let's apply what you learned in the first module. 🤓 🔧